Packs/SymantecEmailSecurity/Integrations/SymantecEmailSecurity/SymantecEmailSecurity.py

```diff
@@ -0,0 +1,1888 @@
+    raw_response = client.list_ioc(args.get("domain", "global"))
+
+    if not argToBoolean(args.get("all_results", False)):
+        limit = arg_to_number(args.get("limit", 50))
+        raw_response = raw_response[:limit]
+
+    table = [
+        {
+            "ID": item.get("iocBlackListId"),
+            "Type": item.get("iocType"),
+            "Value": item.get("iocValue"),
+            "Status": item.get("status"),
+            "Description": item.get("description"),
+            "Email Direction": (
+                EmailDirectionChoices(item["emailDirection"]).name.lower() if item.get("emailDirection") else None
+            ),
+            "Remediation Action": (
+                RemediationActionChoices(item["remediationAction"]).name.lower().replace("_", " ")
+                if item.get("remediationAction")
+                else None
+            ),
+            "Expiry Date": item.get("expiryDate"),
+        }
+        for item in raw_response
+    ]
+
+    readable_output = tableToMarkdown(
+        name="IOC(s)",
+        t=table,
+        headers=[
+            "ID",
+            "Type",
+            "Value",
+            "Status",
+            "Description",
+            "Email Direction",
+            "Remediation Action",
+            "Expiry Date",
+        ],
+        removeNull=True,
+    )
+
+    return CommandResults(
+        outputs_prefix=f"{INTEGRATION_PREFIX}.IOC",
+        outputs_key_field="iocBlackListId",
+        outputs=raw_response,
+        readable_output=readable_output,
+        raw_response=raw_response,
+    )
+
+
+@logger
+def action_ioc_command(client: Client, args: dict[str, Any]) -> CommandResults:
+    """Add, update, delete and renew multiple IOCs.
+
+    Args:
+        client (Client): A session to run HTTP requests to the API.
+        args (dict[str, Any]): Arguments passed down by the CLI to configure the request.
+
+    Returns:
+        CommandResults: Outputs of the command that represent an entry in the warroom.
+    """
+    domain = args.get("domain", "global")
+    action = "IOC" if args["action"] == "upload_ioc_json" else args["action"].upper()
+
+    api_row_action = None
+
+    if action in APIListAction.__members__:
+        api_list_action = APIListAction[action]
+    else:  # action in APIRowActionChoices
+        api_list_action = APIListAction.IOC
+        api_row_action = APIRowActionChoices[action]
+
+    iocs: list[IOC] = []
+
+    if entry_id := args.get("entry_id"):  # Build IOCs from an entry file.
+        if api_row_action:
+            raise DemistoException("The field `entry_id` is only compatible with `action=merge/replace/ioc`.")
+
+        file_entry = demisto.getFilePath(entry_id)
+
+        with open(file_entry["path"], "rb") as handler:
+            content = handler.read()
+
+        iocs = [IOC.from_dict(ioc, api_list_action) for ioc in json.loads(content)]
+    else:  # Build a single IOC from the given params.
+        if action == APIListAction.IOC:
```

Comment:
`merge`, `replace` and `upload_ioc_json` can be used with a file.
This is actually two params combined into one, the orignal was `merge`, `replace`, `ioc` and when choosing , `ioc` you choose an action `add`, `update`, `delete`, `renew`.
It was the Product Managers decision to combine them into one instead of keeping them separated.